product/reports/180_Configuration Management - Instances/010 Amazon - Active VMs.yaml

```diff
@@ -0,0 +1,82 @@
+---
+title: Amazon - Active VMs
+rpt_group: Custom
+rpt_type: Custom
+priority: 20
+db: VmCloud
+cols:
+- name
+- power_state
+include:
+  flavor:
+    columns:
+    - name
+  ext_management_system:
+    columns:
+    - name
+  availability_zone:
+    columns:
+    - name
+  security_groups:
+    columns:
+    - name
+col_order:
+- name
+- flavor.name
+- ext_management_system.name
+- availability_zone.name
+- security_groups.name
+- power_state
+headers:
+- Name
+- Flavor Name
+- Cloud/Infrastructure Provider Name
+- Availability Zone Name
+- Security Group Name
+- Power State
+conditions: !ruby/object:MiqExpression
+  exp:
+    and:
+    - REGULAR EXPRESSION MATCHES:
+        field: VmCloud-vendor
+        value: "/amazon/i"
```

Comment:
Now important is case insensitivity here? Is there not a discreet list of possible values for vendor? The reason I ask is that `REGULAR EXPRESSION MATCHES` is inefficient in that it can only be evaluated in memory. Using an alternative operator (such as `LIKE`, which is case sensitive) can be evaluated in SQL